src/main/java/network/brightspots/rcv/CommonDataFormatReader.java

```diff
@@ -22,28 +22,100 @@
 
 package network.brightspots.rcv;
 
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
 import java.io.File;
+import java.io.FileInputStream;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.logging.Level;
 import javafx.util.Pair;
+import network.brightspots.rcv.CastVoteRecord.CvrParseException;
+import network.brightspots.rcv.RawContestConfig.CvrSource;
 import network.brightspots.rcv.TabulatorSession.UnrecognizedCandidatesException;
 
 class CommonDataFormatReader {
 
+  private final String STATUS_NEEDS_ADJUDICATION = "needs-adjudication";
+  private final String STATUS_YES = "yes";
+  private final String STATUS_NO = "no";
+  private final String STATUS_UNKNOWN = "unknown";
+  private final String BOOLEAN_TRUE = "true";
+  private final String BOOLEAN_FALSE = "false";
```

Comment:
Nit: 3 of these are unused so delete, and the others should probably just be inlined.

Reply:
Yeah I am kinda on the fence here.  My thinking here was to add some names to self-document that 'needs-adjudication' is actually a contest selection 'status' from the CDF spec, and not a magic in-line comment.  You really think it would be clearer to lose them?

Reply:
I see. I think, as a general rule, it's more readable to inline things that are used only once or maybe twice, but I can understand your justification here.

I do still think we should get rid of the three that aren't used at all though (`STATUS_YES`, `STATUS_UNKNOWN`, `BOOLEAN_FALSE`).